src/sdk/types.ts

```diff
@@ -100,6 +100,15 @@ type BaseSDKInitConfig = {
    */
   logLevel?: DiagLogLevel;
 
+  /**
+   * bundleID should only be provided when loading the SDK from CDN through a script tag. It is used to specify a 32
+   * character placeholder string which will then be substituted by our CLI tool when uploading source maps. See
+   * "Including the SDK as a code snippet from CDN" in our README for more details.
+   *
+   * **default**: undefined
+   */
+  bundleID?: string;
```

Comment:
Not sure I follow 100% the replacement process, but this var will always contain the template bundle ID, right? In that case, is it worth making very specific, like `templateBundleID` so that it's clear what is it used for?